fv3core/stencils/fvtp2d.py

```diff
@@ -100,9 +100,6 @@ def __init__(
         self._tmp_q_j = utils.make_storage_from_shape(idx.max_shape, origin)
         self._tmp_fx2 = utils.make_storage_from_shape(idx.max_shape, origin)
         self._tmp_fy2 = utils.make_storage_from_shape(idx.max_shape, origin)
-        self._corner_tmp = utils.make_storage_from_shape(
```

Comment:
Because this is not used outside of the init routine, I changed this to a local variable.